### DISCUSSION TITLE
Is there a way to set default resolution to be coarser for a multi bigwig track in the config.json?

### DISCUSSION THREAD

**Discussion by carolyncaron** [GMOD/jbrowse-components#4597] [Q&A] [answered]
Hello,

I have a multi bigwig track displayed as a single xyplot and one of our researchers pointed out that it is much easier on the eyes if they set the resolution of the track lower using `Score > Resolution > Coarser Resolution`. Is it possible to set this in the config.json as the default (not everyone using our JBrowse2 may be aware of this setting)? 

Here's a screenshot where I copied the track and applied the setting to show what I mean by easier on the eyes:
<img width="615" alt="Screenshot 2024-10-02 at 4 30 57 PM" src="https://github.com/user-attachments/assets/1d112d33-bc08-483f-bc91-0b867dbec83c">

This is a track that's most useful when you see it fully zoomed out with all regions of the assembly.

I haven't seen this documented anywhere and I suspect it might not be as easy as configuring other settings, given that I can continually lower the resolution if I wanted. But I specifically would like it 1 level coarser than the current default, if that makes sense.

**Comment by cmdcolin** [comment, marked as answer]
added a the "resolutionMultiplier" feature in v2.16.1 :) https://github.com/GMOD/jbrowse-components/pull/4619
